src/test/java/seedu/ddd/testutil/VendorUtil.java

```diff
@@ -0,0 +1,41 @@
+package seedu.ddd.testutil;
+
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_SERVICE;
+import static seedu.ddd.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.ddd.logic.commands.AddContactCommand;
+import seedu.ddd.model.contact.vendor.Vendor;
+
+/**
+ * A utility class for Vendor.
+ */
+public class VendorUtil {
+
+    /**
+     * Returns an add command string for adding the {@code contact}.
+     */
+    public static String getAddContactCommand(Vendor vendor) {
+        return AddContactCommand.COMMAND_WORD + " vendor " + getPersonDetails(vendor);
+    }
+
+    /**
+     * Returns the part of command string for the given {@code contact}'s details.
+     */
+    public static String getPersonDetails(Vendor vendor) {
```

Comment:
This should be renamed to `getVendorDetails`.

Reply:
Please also change `getPersonDetails` and `getEditPersonDescriptorDetails` in `ClientUtil`.

Reply:
Can you also change some of the contacts in `TypicalContacts` to `Vendor`s?

This is what I'm referring to:

https://github.com/AY2425S1-CS2103T-F13-3/tp/blob/e94d3524b2a761c9d7e8609ab0895a7378659099/src/test/java/seedu/ddd/testutil/TypicalContacts.java#L27-L44